autoendpoint/src/routers/mod.rs

```diff
@@ -119,7 +119,7 @@ impl RouterError {
         match self {
             RouterError::Adm(e) => e.status(),
             RouterError::Apns(e) => e.status(),
-            RouterError::Fcm(e) => e.status(),
+            RouterError::Fcm(e) => StatusCode::from_u16(e.status().as_u16()).unwrap_or_default(),
```

Comment:
That's annoying, hopefully this will go away at some point (until it maybe happens again) https://github.com/actix/actix-web/pull/3208#issuecomment-2017469785

Reply:
Yeah, to be honest, though, this is just the latest when it comes to this sort of refactoring. `base64` was the last one to do something like this. I expect some other key library to be the next.

And then I get to deal with protobuf3 fun.